src/targeting.js

```diff
@@ -44,6 +45,7 @@ export function getHighestCpmBidsFromBidPool(bidsReceived, highestCpmCallback, a
     // if adUnitBidLimit is set, pass top N number bids
     if (adUnitBidLimit > 0) {
       bucketBids.sort((a, b) => b.cpm - a.cpm);
+      bucketBids = dealPrioritization ? bucketBids(sortByDealAndPriceBucketOrCpm(true)) : bucketBids.sort((a, b) => b.cpm - a.cpm);
```

Comment:
If we're making this sorting type an if/else condition, can we remove the `sort` done on line 47?